qiskit/pulse/library/continuous.py

```diff
@@ -158,14 +158,15 @@ def _fix_gaussian_width(gaussian_samples, amp: float, center: float, sigma: floa
     zeroed_width: Subtract baseline from gaussian pulses to make sure
         $\Omega_g(center \pm zeroed_width/2)=0$ is satisfied. This is used to avoid
         large discontinuities at the start of a gaussian pulse. If unsupplied,
-        defaults to $2*center + 1$ such that $\Omega_g(-1)=0$ and $\Omega_g(2*center + 1)=0$.
+        defaults to $2*(center + 1)$ such that $\Omega_g(-1)=0$ and $\Omega_g(2*(center + 1))=0$.
     rescale_amp: If True the pulse will be rescaled so that $\Omega_g(center)=amp$.
     ret_scale_factor: Return amplitude scale factor.
     """
     if zeroed_width is None:
-        zeroed_width = 2*center + 1
+        zeroed_width = 2*(center + 1)
 
-    zero_offset = gaussian(np.array([zeroed_width/2]), amp, 0, sigma)
+    # compute the offset using ``zeroed_width-1`` (``zeroed_width`` includes both endpoints)
+    zero_offset = gaussian(np.array([(zeroed_width-1)/2]), amp, 0, sigma)
```

Comment:
Why was this changed?

Reply:
My mistake. Fixed in https://github.com/Qiskit/qiskit-terra/pull/4640/commits/8b0c388cfc654b0aa78a1001d770ea1480b5203c.